src/sidebar/services/test/toast-messenger-test.js

```diff
@@ -91,25 +91,25 @@ describe('ToastMessengerService', () => {
       assert.notCalled(fakeStore.addToastMessage);
     });
 
-    it('adds a new error toast message to the store', () => {
-      fakeStore.hasToastMessage.returns(false);
-
-      service.error('boo');
-
-      assert.calledWith(
-        fakeStore.addToastMessage,
-        sinon.match({ type: 'error', message: 'boo' }),
-      );
-    });
-
-    it('does not dismiss the message if `autoDismiss` is false', () => {
```

Comment:
This test was actually not relevant. The `ToastMessenger` is no longer responsible of handling auto-dismising messages, so `fakeStore.removeToastMessage` was never going to be called anyway.

The new test now fails if we don't pass `autoDismiss` to `store.addToastMessage(...)`.